Detector/DetFCChhCalDiscs/compact/Forward_coneCryo.xml

```diff
@@ -0,0 +1,189 @@
+        author="M.Aleksa,J.Faltova,A.Zaborowska"
+        url="no"
+        status="development"
+        version="1.0">
+    <comment>
+      Liquid argon calorimeter design for forward region.
+      Both parts are immersed in the same cryostat.
+      Absorbers have shape of simple discs.
+    </comment>
+  </info>
+
+  <define>
+    <!-- cryostat, common for EMFWD and HFWD -->
+    <constant name="CryoThicknessFront" value="50*mm"/>
+    <constant name="CryoThicknessBack" value="100*mm"/>
+    <constant name="CryoThicknessInner" value="20*mm"/>
+    <constant name="CryoThicknessOuter" value="100*mm"/>
+    <constant name="CryoFwd_front_rmin1" value="FwdCal_rmin1"/>
+    <constant name="CryoFwd_front_rmin2" value="FwdCal_rmin1+CryoThicknessFront*tan(2*atan(exp(-6)))"/>
+    <constant name="CryoFwd_back_rmin1" value="FwdCal_rmin2-CryoThicknessBack*tan(2*atan(exp(-6)))"/>
+    <constant name="CryoFwd_back_rmin2" value="FwdCal_rmin2"/>
+    <constant name="CryoFwd_rmax" value="FwdCal_rmax"/>
+    <constant name="CryoFwd_z1" value="FwdCal_zOffset - FwdCal_dz"/>
+    <constant name="CryoFwd_z2" value="FwdCal_zOffset + FwdCal_dz"/>
+    <constant name="CryoFwd_dz" value="FwdCal_dz"/>
+    <constant name="CryoFwd_zOffset" value="FwdCal_zOffset"/>
+    <!-- space between the cryostat and calorimeter -->
+    <constant name="BathThicknessFront" value="90*mm"/>
+    <constant name="BathThicknessBack" value="50*mm"/>
+    <constant name="BathThicknessOuter" value="50*mm"/>
+    <constant name="BathThicknessMiddle" value="50*mm"/> <!-- in between of EMFWD and HFWD -->
+    <!-- electromegnetic calorimeter: EMFWD -->
+    <constant name="EMFWD_z1" value="CryoFwd_z1 + CryoThicknessFront + BathThicknessFront"/>
+    <constant name="EMFWD_z2" value="17350*mm"/>
+    <constant name="EMFWD_rmin1" value="CryoFwd_front_rmin2 + CryoThicknessInner + BathThicknessFront*tan(2*atan(exp(-6)))"/>
+    <constant name="EMFWD_rmin2" value="EMFWD_z2 * tan(2*atan(exp(-6))) - 2*cm + CryoThicknessInner"/>
+    <constant name="EMFWD_rmax" value="FwdCal_rmax - CryoThicknessOuter - BathThicknessOuter"/>
+    <constant name="EMFWD_lAr_thickness" value="0.1*mm"/>
+    <constant name="EMFWD_Cu_thickness" value="3*cm"/>
+    <constant name="EMFWD_readout_thickness" value="1.2*mm"/>
+    <!-- hadronic calorimeter: HFWD -->
+    <constant name="HFWD_z1" value="EMFWD_z2 + BathThicknessMiddle"/>
+    <constant name="HFWD_z2" value="CryoFwd_z2 - CryoThicknessBack - BathThicknessBack"/>
+    <constant name="HFWD_rmin1" value="HFWD_z1*tan(2*atan(exp(-6))) - 2*cm + CryoThicknessInner"/>
+    <constant name="HFWD_rmin2" value="CryoFwd_back_rmin1 + CryoThicknessInner - BathThicknessBack*tan(2*atan(exp(-6)))"/>
+    <constant name="HFWD_rmax" value="FwdCal_rmax - CryoThicknessOuter - BathThicknessOuter"/>
+    <constant name="HFWD_lAr_thickness" value="0.1*mm"/>
+    <constant name="HFWD_Cu_thickness" value="4.*cm"/>
+    <constant name="HFWD_readout_thickness" value="1.2*mm"/>
+  </define>
```

Comment:
Could you please to change to the naming convention we are using for calorimeters? Not to introduce more and more shortcuts :) EMFWD -> EFCAL, HFWD -> HFCAL. Thanks!